infrastructure/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "docker_image_name" {
```

Comment:
**issue:** replace this with image_tag, something like https://github.com/ioet/time-tracker-v2/blob/255112b0f270ae17de84a20cd7791705dc573f8e/infrastructure/variables.tf#L7